extension.json

```diff
@@ -64,6 +64,11 @@
 			"description": "Sets the external link target (_self for the current tab or _blank for a new tab). (string)",
 			"public": true,
 			"value": "_self"
+		},
+		"RottenLinksUserAgent": {
+			"description": "Overrides the user-agent to use for requests. Defaults to 'RottenLinks, MediaWiki extension (https://github.com/miraheze/RottenLinks), running on <Wiki base URL>'. (string)",
+			"public": true,
+			"value": ""
```

Comment:
Another option here is actually assigning the default here, but instead of the server config, use {server} or something, and a str_replace on that. But I can do that later, this PR looks fine to me now. I'll probably merge a bit later.